test/prism_regression/rescue.rb

```diff
@@ -1,5 +1,60 @@
 # typed: false
 
+begin
```

Comment:
Would you mind adding comments to these examples explaining what's different about each of them so this test file is easier to glance through?

Reply:
Added 👍